libass/ass_drawing.h

```diff
@@ -44,9 +44,12 @@ typedef struct ass_drawing_token {
 typedef struct {
     char *text; // drawing string
     int scale;  // scale (1-64) for subpixel accuracy
-    double pbo; // drawing will be shifted in y direction by this amount
     double scale_x;      // FontScaleX
     double scale_y;      // FontScaleY
+    double pbo; // drawing will be shifted in y direction by this amount
+} ASS_DrawingDesc;
```

Comment:
Bad name... I can't tell the difference between ASS_DrawingDesc and ASS_Drawing.

Reply:
I've already get rid of both structs altogether in next commits.

Reply:
Relevant commits have merged together and there is no longer ASS_DrawingDesc struct.